lib/commands/element.js

```diff
@@ -17,18 +17,20 @@ commands.getAttribute = async function (attribute, el) {
   let atomsElement = this.getAtomsElement(el);
   if (_.isNull(atomsElement)) {
     throw new errors.UnknownError(`Error converting element ID for using in WD atoms: '${el}`);
-  } else {
-    return await this.executeAtom('get_attribute_value', [atomsElement, attribute]);
   }
+  return await this.executeAtom('get_attribute_value', [atomsElement, attribute]);
 };
 
 commands.getText = async function (el) {
   el = util.unwrapElement(el);
+  let result;
   if (!this.isWebContext()) {
-    return await this.proxyCommand(`/element/${el}/text`, 'GET');
+    result = await this.proxyCommand(`/element/${el}/text`, 'GET');
+    return _.isString(result) ? result : JSON.stringify(result);
```

Comment:
Since this is repeated, why not do the `result` setting in the conditional and then return outside
```js
let result;
if (!this.isWebContext()) {
  result = await this.proxyCommand(`/element/${el}/text`, 'GET');
} else {
  let atomsElement = this.useAtomsElement(el);
  result = await this.executeAtom('get_text', [atomsElement]);
}
return _.isString(result) ? result : JSON.stringify(result);
```